Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.13.1-alpine3.21

```

Comment:
The base image `python:3.13.1-alpine3.21` does not exist. Consider using a stable and available base image like `python:3.10-slim` to ensure the image is as thin as possible, as per the checklist requirement.